README.md

```diff
@@ -63,7 +63,7 @@ hash.
 
 ## Linux
 
-- Chez Scheme 9.5.4 or 9.5.5
+- Chez Scheme 9.5.4, 9.5.6 or 9.5.7
```

Comment:
Consider using the Oxford comma here and in the sections for Mac and Windows.

Reply:
Done!